core/src/main/java/org/apache/calcite/sql/type/ReturnTypes.java

```diff
@@ -645,6 +645,13 @@ public static SqlCall stripSeparator(SqlCall call) {
   public static final SqlReturnTypeInference TO_ARRAY =
       ARG0.andThen(SqlTypeTransforms.TO_ARRAY);
 
+  /**
+   * Same as {@link #TO_ARRAY} but returns with nullability if any of the
```

Comment:
Would you mind updating this javadoc to match the format of other `*_NULLABLE` return types? Here is `DATE` and `DATE_NULLABLE` as an example:

```
  /**
   * Type-inference strategy whereby the result type of a call is DATE.
   */
  public static final SqlReturnTypeInference DATE =
      explicit(SqlTypeName.DATE);

  /**
   * Type-inference strategy whereby the result type of a call is nullable
   * DATE.
   */
  public static final SqlReturnTypeInference DATE_NULLABLE =
      DATE.andThen(SqlTypeTransforms.TO_NULLABLE);
```